web/Database/PropertyDbModel.cs

```diff
@@ -23,4 +23,6 @@ public class PropertyDbModel
     public DateTime? CreationTime { get; set; }
     public int? Rent { get; set; }
     public string? Description { get; set; }
+    public string? Lat { get; set; } // latitude
+    public string? Lon { get; set; } // longitude
```

Comment:
Feels odd to store them as strings. `FLOAT` (C#'s `double` type) should work, or you could use `DECIMAL`.

Azure probably returns them as strings to make sure they keep precision (lots of decimal places), but we don't need it to be super precise.